slither/solc_parsing/declarations/contract.py

```diff
@@ -277,13 +277,20 @@ def _parse_contract_items(self) -> None:
             elif item[self.get_key()] == "UsingForDirective":
                 self._usingForNotParsed.append(item)
             elif item[self.get_key()] == "ErrorDefinition":
-                self._customErrorParsed.append(item)
+                self._customErrorsNotParsed.append(item)
             elif item[self.get_key()] == "UserDefinedValueTypeDefinition":
                 self._parse_type_alias(item)
             else:
                 raise ParsingError("Unknown contract item: " + item[self.get_key()])
         return
 
+    def parse_type_alias(self) -> None:
+        # We keep parse_ in the name just to keep the naming convention even if we already parsed them initially.
+        # Here we only update the current contract type_aliases_as_dict with the fathers' values
+        # It's useful to keep using the same pattern anyway as we know all the fathers have been analyzed
+        for father in self._contract.inheritance_reverse:
```

Comment:
Is reversed right? I think the most recent ancestor should take priority

Reply:
type alias declarations can not be shadowed or you get an identifier already declared error so i believe you get the same result